test/src/unit-cppapi-query-condition-sets.cc

```diff
@@ -202,6 +207,21 @@ TEST_CASE_METHOD(
   check_read(qc, [](const QCSetsCell& c) { return (c.d == 1 || c.d == 5); });
 }
 
+TEST_CASE_METHOD(
+    CPPQueryConditionFx,
+    "IN - Enumeration",
+    "[query-condition][set][basic][enumeration]") {
+  auto type = GENERATE(
+      TestArrayType::DENSE, TestArrayType::SPARSE, TestArrayType::LEGACY);
+  auto serialize = SERIALIZE_TESTS();
+  create_array(type, serialize);
+
+  std::vector<std::string> values = {"wilma", "betty"};
+  auto qc = QueryConditionExperimental::create(ctx_, "attr6", values);
```

Comment:
Why does this not have a `TILEDB_IN` argument?

Reply:
TILEDB_IN is the default: https://github.com/TileDB-Inc/TileDB/blob/70ad7bb131f6edd3b9d75e139efe40069d302c74/tiledb/sm/cpp_api/query_condition_experimental.h#L64

Reply:
Looking back, this seemed a bit weird since its just called "create" so having it assume set operations for a std::vector argument seemed a bit lax on my part. I removed the default parameter and updated all the call sites to pass `TILEDB_IN`. That'll make things less awkward if we ever have another QueryCondition thing that accepts vectors.